src/main/resources/logback-spring.xml

```diff
@@ -55,6 +55,11 @@
     <appender-ref ref="consoleAppender"/>
   </logger>
 
+  <logger name="org.flywaydb.core.internal.sqlscript.DefaultSqlScriptExecutor" additivity="false" level="WARN">
```

Comment:
As I'm refreshing the database with flyway after each test class, there was a lot of noise in the logs so turned it down.

Reply:
I noticed that we have started getting
```
2021-04-27 13:07:26.999  INFO 896483 --- [           main] o.f.c.i.s.DefaultSqlScriptExecutor       : 1 rows affected |  
```
recently all over the place, haven't decided yet if it is helpful though.
Should this have gone in a `logback-spring-test.xml` so it is only applied for tests?

Reply:
Well all I'm seeing in this project are pages and pages of `2021-04-28 16:49:04.538  INFO 69012 --- [    Test worker] o.f.c.i.s.DefaultSqlScriptExecutor       : 1 rows affected |  `, so not very useful.  `2021-04-28 16:49:04.403  INFO 69012 --- [    Test worker] o.f.core.internal.command.DbMigrate      : Migrating schema "public" to version "1.0 - create tables" |` is much more useful.

I considered creating a test logback xml but it was just more stuff to think about...